### PR TITLE
Adding Release notes for 2.10.0

### PR DESCRIPTION
**Pull request by poojaSprophecy**

### CODE REVIEW

File: docs/metadata/personalAccessTokens.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Create and Manage API tokens from the Access Tokens tab in [Settings](https://app.prophecy.io/metadata/settings)) page. You can create, delete, check usage, expiry etc of the tokens from this page. Please copy the token you have generated, as this will not be displayed on Prophecy UI again.
+You can use this token for Accessing any Read/Write APIs to access Metadata, Lineage , etc.
+
+Please note, These tokens are per user and will inherit all the Access from that user.
+
+![AccessToken](./img/PAT.gif)
+
+:::info
+**Coming Soon**
+We will soon launch a detailed Documentation and a playground to try these APIs
```
Comment (nadeesh-garg) [inline docs/metadata/personalAccessTokens.md:27]: We are building*
Thank you for your patience?

**Review by nadeesh-garg — APPROVED** [review]

File: docs/release_notes/mar2023.md [hunk 2]
```diff
@@ -8,6 +8,49 @@ tags:
   - changelog
 ---
 
+## 2.10.0.0 (Mar 31, 2023)
```
Comment (atbida) [inline docs/release_notes/mar2023.md:11]: This should be linked from the /release_notes/ page, right?

File: docs/release_notes/mar2023.md [hunk 3]
```diff
@@ -8,6 +8,49 @@ tags:
...
+
+- Prophecy Python libs version: 1.4.8
+- Prophecy Scala libs version: 7.0.16
+
+### Updates {#UpdatesRelease2100}
+
+#### Support for Personal Access Tokens
```
Comment (atbida) [inline docs/release_notes/mar2023.md:18]: Was the API accessible previously without PATs? Is this a security improvement? Would this be correct?
"The Prophecy metadata API is now accessible only via Personal Access Token"

Perhaps we want to "Introduce" the metadata API as though it's the first time this API is accessible, and it's a very secure approach, like this: 
"Now Prophecy users can access their metadata using the Prophecy API. Just create a Personal Access Token (PAT) (link). One popular use case is to access the Prophecy metadata through automated scripts and integrate this data into an existing data visualization system. Use the Prophecy API to create a single pane of glass interface for data stewards and engineers to see pipelines, datasets, Lineage, etc. Please see [here] for more details on this feature."

File: docs/release_notes/mar2023.md [hunk 4]
```diff
@@ -8,6 +8,49 @@ tags:
...
+A Personal Access Token(PAT) is the token which customer can use for authentication to our API servers.
+This will enable customers to use PAT in their scripts and integrate Prophecy seamlessly to their data visualization ecosystem.
+They would be able to access Prophecy metadata through APIs and create a single pane of glass interface for their data stewards and engineers to see pipelines, datasets, Lineage, etc.
+Please see [here](./../metadata/personalAccessTokens) for more details on this feature.
+
+### Improvements {#ImprovementsRelease2100}
+
+#### Search box on available clusters
+
+To help Users find there clusters faster, we have added Search capability when listing down the clusters in Attach cluster screens. Please see below video for example.
```
Comment (atbida) [inline docs/release_notes/mar2023.md:29]: their

File: docs/release_notes/mar2023.md [hunk 5]
```diff
@@ -8,6 +8,49 @@ tags:
...
+
+![SearchCluster](img/ClusterSearch.gif)
+
+#### Performance Improvements for Projects with large Number of Entities
+
+With this release, we have Optimized the Project Browser to handle Large Projects (Project With of Pipelines/Datasets in orders of 500s).
```
Comment (atbida) [inline docs/release_notes/mar2023.md:35]: With this release, we have optimized the Project Browser to handle Projects with more than 500 Pipelines/Datasets. User should not see delays and should be able to seamlessly browse Entities in large Projects.

**Pull request merged**